app/models/patient.rb

```diff
@@ -1054,6 +1054,7 @@ def household
 
   # Single place for calculating the end of monitoring date for this subject.
   def end_of_monitoring
+    return '' if isolation
```

Comment:
Should this be `nil`?

Reply:
[f82c651073e2f2f5e32b2a0aff27484897925e95](https://github.com/SaraAlert/SaraAlert/pull/1192/commits/f82c651073e2f2f5e32b2a0aff27484897925e95)

Yep! Didn't realize there were places in the app where the check for nil happens and replaces with a blank before hitting the React component or Export.